test/unit/fixtures.py

```diff
@@ -1,9 +1,14 @@
 import json
 import os
+import re
 import sys
 
 FIXTURES_DIR = sys.path[0] + "/test/fixtures"
 
+# This regex is useful for finding individual underscore characters,
+# which is necessary to allow us to use underscores in URL paths.
+PATH_REPLACEMENT_REGEX = re.compile(r"(?<!_)_(?!_)")
```

Comment:
This is necessary for `lke/clusters/18881/control_plane_acl` fixture since the `_`s were being replaced with `/`s.